clang/include/clang/AST/CXXInheritance.h

```diff
@@ -268,7 +268,7 @@ struct UniqueVirtualMethod {
 /// subobject in which that virtual function occurs).
 class OverridingMethods {
   using ValuesT = SmallVector<UniqueVirtualMethod, 4>;
-  using MapType = llvm::MapVector<unsigned, ValuesT>;
+  using MapType = llvm::SmallMapVector<unsigned, ValuesT, 16>;
```

Comment:
It looks like this is nested inside another MapVector. I'd drop this one if it doesn't have significant effect.